UnitsNet.Tests/CustomCode/MassTests.cs

```diff
@@ -57,9 +57,9 @@ public class MassTests : MassTestsBase
 
         protected override double ShortHundredweightInOneKilogram => 0.022046226218487758;
 
-        protected override double EarthMassesInOneKilogram => 1.0 / 5.9722E+24;
+        protected override double EarthMassesInOneKilogram => 1.6744248350691500000000000E-25;
 
-        protected override double SolarMassesInOneKilogram => 1.0 / 1.98847e30;
+        protected override double SolarMassesInOneKilogram => 5.0289921396852900000000000E-31;
 
         protected override double SolarMassesTolerance => 0.1;
```

Comment:
I just realized, this is a pretty poor tolerance value. I mean, are you sure you are happy with converting between say solar and earth masses and getting worst case up to +/- 0.1 in absolute error? That sounds like a big error to me.

I think this could be a good case for explicitly mapping direct conversions between solar and earth mass, since those units are soo much bigger than the base unit Gram, which all Mass conversions convert via.

We just merged a new feature that allows dynamic conversion functions https://github.com/angularsen/UnitsNet/pull/588. See for example how [UnitConverter.RegisterDefaultConversions()](https://github.com/angularsen/UnitsNet/pull/588/files#diff-e8f14ed18a8a392dd1b1e9815e3cc303R35) maps all the built-in conversions, such as Kilogram -> Gram and Gram -> Kilogram, etc.. 

See also this [test case on how to add conversion functions between third party units](https://github.com/angularsen/UnitsNet/pull/588/files#diff-a4784d555ebd512185a04efef66de34aR99) .

This is all very new and not done before, but I think you can plug in your conversion functions in the [UnitConverter static constructor](https://github.com/angularsen/UnitsNet/pull/588/files#diff-f8c5cdc547fba4b91a639bd839e80997R48). Here we can add direct conversion functions for certain unit conversions that suffer from poor precision, such as EarthMass -> SolarMass and the other way. Just put them in a method `private static void RegisterDirectConversions() {}` and call it from `UnitConverter` static constructor.

Reply:
No I'm not happy with it, but it was the only way to make it compile. And I don't know enough of the framework to figure out how to do better.

I will try the UnitConverter. But I also thought of an alternative. How about a "StellarMass" quantity with megatons as the baseunit?

Reply:
Ok so I resolved the tolerance issue..... by doing nothing. Well I actually corrected a typo in a value. That was the reason it didn't work. So with my low tolerance override I was just masking the problem. I thought it was related to the big difference with the solar mass and a gram.

Reply:
Hehe :-) Easy thing to do! I'll review this now.